### ISSUE TITLE
Check for misspelled rel values; e.g., styleshet

### ISSUE THREAD

**Issue by Does-a-thingy** [validator/validator#1102]
URL being validated or code to reproduce error:

```html
<!doctype html>
<title>Validation lesson</title>
<link rel="styleshet" href="/css/profile1.css">

<h1>HTML Validators</h1>
<p>Validators are free tools that check your code follows the W3C rules.
```

There was no error message for the rel="styleshet" to tell me that was incorrect.

**Comment by sideshowbarker** [comment]
We no longer check the contents of `rel` values against anything — because there’s basically a nearly unbounded set of `rel` values in use. But I imagine what I could do to help catch misspellings like this is, implement some Levenshtein-distance check against the list of ~20 known values that are defined in the HTML spec, and if that shows a close match, report a warning.
